src/Shared/FileMatcher.cs

```diff
@@ -2306,23 +2301,25 @@ private static int SkipSlashes(string aString, int startingIndex)
         /// </summary>
         private static bool IsDriveEnumeratingWildcardPattern(string directoryPart, string wildcardPart)
```

Comment:
Recent change questions:
1. I know Ladi implemented something similar in his PR, but the code seemed further away and I needed a slightly separate way to ensure that drive enumeration could be detected using the `fixedDirectoryPart `and `wildcardDirectoryPart`. Is there a way to optimize this code-wise and performance-wise? I account for \** and <drive>:<any number of slashes>** with this function.

Reply:
Can you link to that other implementation?

Reply:
Yes, it's the change in ItemSpec.cs here: https://github.com/dotnet/msbuild/pull/5669/files